WalletWasabi.Fluent/Models/UI/UiClipboard.cs

```diff
@@ -9,6 +10,20 @@ public partial class UiClipboard
 {
 	public async Task<string> GetTextAsync() => await ApplicationHelper.GetTextAsync();
 
+	public async Task<string?> TryGetTextAsync()
+	{
+		try
+		{
+			return await GetTextAsync();
+		}
+		catch (Exception ex)
+		{
+			Logger.LogError(ex);
+		}
+
+		return null;
+	}
```

Comment:
No. This is used because I've noticed there are times when `GetTextAsync()` crashes inside an Avalonia method.
I decided to just ignore the exception and retry later.